spring-cloud-skipper-client/src/main/java/org/springframework/cloud/skipper/client/DefaultSkipperClient.java

```diff
@@ -238,6 +240,43 @@ public Release scale(String releaseName, ScaleRequest scaleRequest) {
 		return resourceResponseEntity.getBody();
 	}
 
+	@Override
+	public String getFromActuator(String releaseName, String appName, String appId, String endpoint) {
+
+		Map<String, Object> uriVariables = new HashMap<>();
+		uriVariables.put("releaseName", releaseName);
+		uriVariables.put("appName", appName);
+		uriVariables.put("appId", appId);
+
+		ResponseEntity<String> resourceResponseEntity =
+				restTemplate.exchange(
+						baseUri + "/release/actuator/{releaseName}/{appName}/{appId}?endpoint=" + endpoint,
```

Comment:
**[UNRELATED]**: I would expect the `releaseName` to come after `release` rather than `actuator` in the path. Such as `/release/{releaseName}/(actuator|scale|etc)`.

I see the other apis do the same. Obviously not asking for a change here, just wanting to understand if it was intentional etc..